WarehouseManagement.Api/Controllers/DifferenceTypeController.cs

```diff
@@ -1,11 +1,13 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
 using System.Security.Claims;
 using WarehouseManagement.Core.Contracts;
 using WarehouseManagement.Core.DTOs.DifferenceType;
 using static WarehouseManagement.Common.MessageConstants.Keys.DifferenceTypeMessageKeys;
 
 namespace WarehouseManagement.Api.Controllers;
 
+[Authorize]
```

Comment:
dont we need this for all controllers?

Reply:
Yes I put it just to test it. We should discuss which routes will be accessible only by authorized users.